<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>이  름</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>암  호</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>암호재입력</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>2-20 글자</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>3-20글자, 중복안됨</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pw1">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>8-20글자, 문자/숫자/특수문자</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pw2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>암호와 동일하게 입력</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_pw1</tabstop>
  <tabstop>lineEdit_pw2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_membership()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_name</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_checkName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_id</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_checkId()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_pw1</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_checkPw1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_pw2</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_checkPw2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_membership()</slot>
  <slot>slot_checkId()</slot>
  <slot>slot_checkPw1()</slot>
  <slot>slot_checkPw2()</slot>
  <slot>slot_checkName()</slot>
  <slot>slot_labelClear()</slot>
 </slots>
</ui>
